core/src/commands/area.cpp

```diff
@@ -215,10 +215,12 @@ void AOClient::cmdGetAreas(int argc, QStringList argv)
     Q_UNUSED(argv);
 
     QStringList l_entries;
-    l_entries.append("== Area List ==");
+    l_entries.append("\n== Currently Online: " + QString::number(server->m_player_count));
```

Comment:
```suggestion
    l_entries.append("\n== Currently Online: " + QString::number(server->m_player_count)+ " ==");
```
gotta keep that symmetry